pop-api/src/v0/assets/mod.rs

```diff
@@ -91,13 +91,11 @@ const TRANSFER_APPROVED: u8 = 25;
 /// Move some assets from the sender account to another, keeping the sender account alive.
 #[inline]
 pub fn transfer_keep_alive(id: AssetId, target: AccountId, amount: Balance) -> Result<()> {
-	ChainExtensionMethod::build(u32::from_le_bytes([
-		V0,
-		DISPATCH,
+	build_dispatch(
 		ASSETS,
 		// E.D. is always respected with transferring tokens via the API.
```

Comment:
Be good to place this comment above the build dispatch call, only so it fits on a single line.